src/interfaces/StripeOptions.ts

```diff
@@ -1,10 +1,5 @@
-import * as ProxyAgent from 'https-proxy-agent';
+import Stripe from 'stripe';
```

Comment:
Can you do `import { StripeConfig } from 'stripe';` and update the below

Reply:
I think that is better, too :)

https://github.com/stripe/stripe-node/blob/9d817130dd22312f7d1905fafccca45610df0e46/types/lib.d.ts#L7-L11

`StripeConfig` depends on `namespace Stripe`, cant export only `StripeConfig` :'(

Reply:
Interesting! Okay I guess I can live with that 😆